### DISCUSSION TITLE
Classification

### DISCUSSION THREAD

**Discussion by Zubbyz01** [semiautomaticgit/SemiAutomaticClassificationPlugin#221] [General]
Please can anybody give more details on choosing the right training sites especially how to differentiate between, built up and bareland and also natural vegetation and cropland..thanks

**Comment by semiautomaticgit** [comment]
Hello @Zubbyz01 , you should analyze spectral signatures and the spectral distance between them.
You can calculate spectral distances using the spectral plot
https://semiautomaticclassificationmanual.readthedocs.io/en/latest/spectral_signature_plot.html#spectral-distances

In general, you should try to collect training areas that have high spectral distance between different classes. Of course, some materials are spectrally similar, so you should also investigate input data, for instance using images acquired at different dates, and analyze the differences between spectral signatures
